pkgs/test_api/lib/src/backend/runtime.dart

```diff
@@ -89,7 +89,7 @@ final class Runtime {
   final Compiler defaultCompiler;
 
   /// All the supported compilers for this runtime.
-  final List<Compiler> supportedCompilers;
+  final Set<Compiler> supportedCompilers;
```

Comment:
This is also breaking (this class is also exported by backend.dart)

Reply:
Ugh. k.

Reply:
This one would break some things

Reply:
I don't think this will break anything. Only `Iterable` interface methods are used on this field, and only from `test_core`

Reply:
It is breaking because this field is used in initializing formal parameters, and the constructor is used internally at least and possibly also in flutter but not sure on that.

Reply:
We could possibly change the constructor to take an iterable and call `toSet` but that is less than ideal probably